argocd/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM --platform=$BUILDPLATFORM alpine as build
```

Comment:
I don't understand why BUILDPLATFORM is needed in this Dockerfile, can you elaborate?

Reply:
there is no need to do that, you're right